Source/WebKit/Shared/API/c/WKContextMenuItemTypes.h

```diff
@@ -131,6 +131,7 @@ enum {
     kWKContextMenuItemTagAddHighlightToNewQuickNote,
     kWKContextMenuItemTagRevealImage,
     kWKContextMenuItemTagTranslate,
+    kWKContextMenuItemTagSwapCharacters,
```

Comment:
Please add new enum values to the end, to preserve bincompat (since this is an SPI value)